src/main/io/gps/gps.c

```diff
@@ -243,13 +254,63 @@ typedef struct ubxCfgNav5_s {
     uint16_t staticHoldMaxDist;
     uint8_t utcStandard;
     uint8_t reserved1[5];
-} ubxCfgNav5_t;
+} ubxCfgNav5m8_t;
+
+typedef struct ubxCfgNav5m10_s {
+    uint16_t mask;
+    uint8_t dynModel;
+    uint8_t fixMode;
+    int32_t fixedAlt;
+    uint32_t fixedAltVar;
+    int8_t minElev;
+    uint8_t drLimit;
+    uint16_t pDOP;
+    uint16_t tDOP;
+    uint16_t pAcc;
+    uint16_t tAcc;
+    uint8_t staticHoldThresh;
+    uint8_t dgnssTimeout;
+    uint8_t cnoThreshNumSVs;
+    uint8_t cnoThresh;
+    uint8_t reserved0[2];
+    uint16_t staticHoldMaxDist;
+    uint8_t utcStandard;
+    uint8_t reserved1[5];
+} ubxCfgNav5m10_t;
+
```

Comment:
Should we only include values that we actually use, and comment out the others?

Reply:
No, the payload needs a fixed size and structure and ublox says
"all unused parameters must be 0"

so we need the full struct sadly